styles.css

```diff
@@ -0,0 +1,168 @@
+/* global styles */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-weight: 400;
+  font-family: "Nunito Sans", sans-serif;
+  --brand-primary: #0768ac;
+  --brand-secondary: #03c180;
+  --bg_default: #ffffff;
+  --bg_body: #f0f9ff;
+  --lines-color: #dddddd;
+  --body-text: #333333;
+  --heart-color: #dc143c;
```

Comment:
here you are defining this variables inside all elements. useuelly we define variables inside the root:

```
:root { 
--brand-primary: #0768ac
...
}
```